src/test/keyutilstest.cpp

```diff
@@ -142,6 +142,47 @@ TEST_F(KeyUtilsTest, KeyNameNotation) {
               KeyUtils::guessKeyFromText("cb"));
     EXPECT_EQ(mixxx::track::io::key::C_MINOR,
               KeyUtils::guessKeyFromText("b#"));
+
+    // Rapid Evolution test cases
+    EXPECT_EQ(mixxx::track::io::key::A_MINOR,
+            KeyUtils::guessKeyFromText("Am"));
+    EXPECT_EQ(mixxx::track::io::key::A_MINOR,
+            KeyUtils::guessKeyFromText("08A"));
+    EXPECT_EQ(mixxx::track::io::key::B_FLAT_MINOR,
+            KeyUtils::guessKeyFromText("A#m"));
+    EXPECT_EQ(mixxx::track::io::key::B_FLAT_MINOR,
+            KeyUtils::guessKeyFromText("Bbm"));
+    EXPECT_EQ(mixxx::track::io::key::A_MAJOR,
+            KeyUtils::guessKeyFromText("11B"));
+    EXPECT_EQ(mixxx::track::io::key::A_FLAT_MAJOR, // ionian
+            KeyUtils::guessKeyFromText("G#+50"));
+    EXPECT_EQ(mixxx::track::io::key::A_FLAT_MAJOR, // ionian
+            KeyUtils::guessKeyFromText("Ab +50"));
+    EXPECT_EQ(mixxx::track::io::key::A_FLAT_MAJOR, // ionian
+            KeyUtils::guessKeyFromText("Ab +50cents"));
+    EXPECT_EQ(mixxx::track::io::key::A_FLAT_MAJOR, // ionian
+            KeyUtils::guessKeyFromText("04B +50cents"));
+    EXPECT_EQ(mixxx::track::io::key::A_FLAT_MAJOR, // ionian
+            KeyUtils::guessKeyFromText("G#-50"));
+    EXPECT_EQ(mixxx::track::io::key::A_FLAT_MAJOR, // ionian
+            KeyUtils::guessKeyFromText("Ab -50"));
+    EXPECT_EQ(mixxx::track::io::key::A_FLAT_MAJOR, // ionian
+            KeyUtils::guessKeyFromText("Ab -50cents"));
+    EXPECT_EQ(mixxx::track::io::key::A_FLAT_MAJOR, // ionian
+            KeyUtils::guessKeyFromText("04B -50cents"));
+    EXPECT_EQ(mixxx::track::io::key::A_FLAT_MAJOR, // ionian
+            KeyUtils::guessKeyFromText("    4b    -50   cents    "));
+    // Mixxx does not allow this but Rapid Evolution
+    // EXPECT_EQ(mixxx::track::io::key::A_FLAT_MAJOR, // ionian
+    //          KeyUtils::guessKeyFromText("    g  #    -    50   cents    "));
+    // EXPECT_EQ(mixxx::track::io::key::A_FLAT_MAJOR, // ionian
+    //          KeyUtils::guessKeyFromText("    g  #    +    50   cents    "));
```

Comment:
Your comment is at the disabled test. Mixxx fails to pars that because of too many blanks. I do not consider the test data as realistic and failing is OK or even desired. 

The tuning is trimmed away, because Mixxx does not make use of it right now. 

The next iteration of making Mixxx tunig aware is to store the tuning information in the database. This is implemented here: 
https://github.com/mixxxdj/mixxx/pull/11096 From this we can introduce the next iteration of making it tuning aware.

Reply:
Yes thats exactly what I mean, if rapid evolution allows / exports data with that many blanks, we should accept it. Whats your reasoning for concluding that its not realistic?

Stripping the actual tuning information is fine for now imo.

Reply:
This is artificial data for a full test coverage of the Rapid Evolution key parser. No one exports the data like that.
Since the Mixxxx parser works differently this test is IMHO pointless for us.